heron/cli/src/python/argparser.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python2.7
```

Comment:
I see no need to run this module as a separate Python program. Please remove this shebang.